### ISSUE TITLE
Feature request: Preserve code highlights after closing tabs or restarting IntelliJ

### ISSUE THREAD

**Issue by daalla** [huoguangjin/MultiHighlight#40]
Hi all,

I'd like to propose an enhancement for the plugin. The ability to maintain code highlights even after closing tabs or restarting the IDE would significantly benefit those of us who tackle extensive debugging tasks over multiple sessions.

If there's room for collaboration on this feature, I'd be keen to explore how I could contribute to its development.

Any feedback or thoughts on this would be greatly appreciated.

**Comment by velios** [comment]
Exactly. The ability to save highlighting and restore from a file(or any another way) could help explore different domains within a large application
